CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.34.0
+------------------------------
+*April 12, 2022*
+
+### Updated
```

Comment:
Can you change this to `#changed` https://keepachangelog.com/en/1.0.0/

Reply:
Will do!